clients/cli/src/common.rs

```diff
@@ -71,3 +72,11 @@ pub async fn process_transaction(
         ))
     }
 }
+
+pub async fn get_account_owner(
+    account: &Pubkey,
+    rpc_client: Arc<RpcClient>,
```

Comment:
nit: better to just accept `&RpcClient` directly rather than cloning an arc to use the function